[2023-12-12T01:31:07.720+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_openaq.load_measures_data manual__2023-12-12T01:29:55.920201+00:00 [queued]>
[2023-12-12T01:31:07.724+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_openaq.load_measures_data manual__2023-12-12T01:29:55.920201+00:00 [queued]>
[2023-12-12T01:31:07.724+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 4
[2023-12-12T01:31:07.730+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_measures_data> on 2023-12-12 01:29:55.920201+00:00
[2023-12-12T01:31:07.733+0000] {standard_task_runner.py:57} INFO - Started process 298 to run task
[2023-12-12T01:31:07.740+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_openaq', 'load_measures_data', 'manual__2023-12-12T01:29:55.920201+00:00', '--job-id', '366', '--raw', '--subdir', 'DAGS_FOLDER/dag_openaq.py', '--cfg-path', '/tmp/tmp2rwnc1_w']
[2023-12-12T01:31:07.745+0000] {standard_task_runner.py:85} INFO - Job 366: Subtask load_measures_data
[2023-12-12T01:31:07.783+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_openaq.load_measures_data manual__2023-12-12T01:29:55.920201+00:00 [running]> on host 2be5b1614f4e
[2023-12-12T01:31:07.825+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_openaq' AIRFLOW_CTX_TASK_ID='load_measures_data' AIRFLOW_CTX_EXECUTION_DATE='2023-12-12T01:29:55.920201+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-12T01:29:55.920201+00:00'
[2023-12-12T01:31:07.827+0000] {main.py:140} INFO - Archivo de configuracion leido correctamente.
[2023-12-12T01:31:07.827+0000] {utils.py:25} INFO - Obteniendo datos de https://api.openaq.org/v2/measurements...
[2023-12-12T01:31:08.410+0000] {utils.py:73} INFO - Conexi√≥n a la base de datos establecida exitosamente.
[2023-12-12T01:31:10.768+0000] {utils.py:90} INFO - Cargando datos en la tabla stg_measurements...
[2023-12-12T01:31:12.516+0000] {utils.py:98} INFO - Datos cargados exitosamente en la tabla stg_measurements.
[2023-12-12T01:31:12.519+0000] {main.py:166} INFO - Ejecutando query CALL nicolasmvinciguerra_coderhouse.petl_load_fact_measure();...
[2023-12-12T01:31:13.148+0000] {main.py:168} INFO - Se ejecuto correctamente la query: CALL nicolasmvinciguerra_coderhouse.petl_load_fact_measure();
[2023-12-12T01:31:13.422+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-12T01:31:13.454+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_openaq, task_id=load_measures_data, execution_date=20231212T012955, start_date=20231212T013107, end_date=20231212T013113
[2023-12-12T01:31:13.505+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-12T01:31:13.524+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
